src/modules/blogs/blogs.controller.ts

```diff
@@ -11,11 +11,19 @@ export class BlogController {
   constructor(private readonly blogService: BlogService) {}
 
   @Post()
-  @UseGuards(SuperAdminGuard)
+  // @UseGuards(SuperAdminGuard)
```

Comment:
Why comment this out. If the guard is not needed. Please remove entirely

Reply:
Done.